Polygon_mesh_processing/examples/Polygon_mesh_processing/corefinement_parallel_union_meshes.cpp

```diff
@@ -10,6 +10,7 @@
 
 typedef CGAL::Exact_predicates_inexact_constructions_kernel Kernel;
 typedef Kernel::Point_3 Point_3;
+typedef Kernel::Aff_transformation_3 Aff_transformation_3;
```

Comment:
why changing it in an example?

Reply:
It is not about inclusion order. I only  saw that we mix   `Class<Kernel>`  and `Kernel::Class`  which makes no sense to me even if they are equivalent,

Reply:
my point is why replacing in "user code" a free function call by a functor call.